README_en.md

```diff
@@ -0,0 +1,66 @@
+# Coding Dojo
```

Comment:
Ainda vai fazer alguma tradução neste arquivo? Se não for, assim que arrumar o outro arquivo, já aceito a PR!